getting_started/access_your_services.adoc

```diff
@@ -113,6 +113,74 @@ Alternatively, you can use a custom domain:
 $ oc expose service <service-name> -l route=external --name=<custom-route-name> --hostname=<custom-domain>
 ----
 
+[[expose-tcp-services]]
+== Expose TCP Services
+
+{product-title} xref:../architecture/networking/routes.adoc[routes] expose applications by proxying traffic through HTTP/HTTPS(SNI)/TLS(SNI)
```

Comment:
We need to include the section ID at the top of the page.

xref:../architecture/networking/routes.adoc#architecture-core-concepts-routes[routes]

xref:../architecture/core_concepts/pods_and_services.adoc#architecture-core-concepts-pods-and-services[pods and services]

Reply:
That seems very odd...is it just because I'm `xref`ing the `adoc` version?

Reply:
@wgordon17 Yeah, for some reason the missing section IDs cause the builds to go wonky.

Reply:
Weird 🤷‍♂️

Reply:
I know :-1: